Wikipedia/Legacy Data Migration/WMFLegacyImageDataMigration.swift

```diff
@@ -23,20 +23,8 @@ public class WMFLegacyImageDataMigration : NSObject {
     /// Image controller where data will be migrated.
     let imageController: WMFImageController
 
-    /// List of saved pages which is saved when the tasks are finished processing.
```

Comment:
this was mostly a result of making `userDataStore` a persistent property on `MWKDataStore`, which obviated the need for a separate saved page list in this class and a lot of the logic related to "transferring" the `didMigrateImageData` flags over to the "main" saved page list.